### PR TITLE
Added dbt description toggle docs

### PR DESCRIPTION
**Pull request by OnkarVO7**
### Describe your changes:

Added dbt description toggle docs

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: Ingest Descriptions from dbt
+slug: /connectors/ingestion/workflows/dbt/ingest-dbt-descriptions
+---
+
+# Ingest Descriptions from dbt
+
+Ingest the descriptions from dbt `manifest.json` or `catalog.json` file into OpenMetadata tables and columns.
+
+{% note %}
+By default descriptions from `manifest.json` will be imported. Descriptions from `catalog.json` will only be updated if `catalog.json` file is passed.
```
Comment (pmbrull) [inline openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md:11]: maybe silly question: can we have descriptions in both manifest and catalog? If so, which one are we persisting?

Comment (OnkarVO7) [reply openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md:11]: We can have descriptions in both files
If only manifest file is passed descriptions from manifest will be updated
If manifest and catalog both are passed descriptions from catalog will be updated

File: openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{% /note %}
+
+
+## Overriding the existing table and column descriptions
+
+When the `Update Descriptions` toggle is enabled during the configuration of dbt ingestion, existing descriptions of tables and columns will be overwritten with the dbt descriptions.
```
[the remark this answers is not in the document]
Comment (OnkarVO7) [reply openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md:17]: Yes I will I add that

File: openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md (hunk 1 above)
Comment (pmbrull) [reply openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md:11]: then let's add this info here

Comment (OnkarVO7) [reply openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md:11]: Added the info

**Review by pmbrull — APPROVED** [review]

**Pull request merged**